packages/pwa-kit-react-sdk/src/ssr/server/opentelemetry-server.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {NodeTracerProvider} from '@opentelemetry/sdk-trace-node'
+import {SimpleSpanProcessor} from '@opentelemetry/sdk-trace-base'
+import {B3Propagator} from '@opentelemetry/propagator-b3'
+import {Resource} from '@opentelemetry/resources'
+import {SemanticResourceAttributes} from '@opentelemetry/semantic-conventions'
+import {propagation} from '@opentelemetry/api'
+
+const SERVICE_NAME = 'pwa-kit-react-sdk'
+
+let provider = null
+
+/**
+ * Initialize OpenTelemetry tracing for server-side rendering
+ * @returns {NodeTracerProvider|null} The initialized provider or null if initialization failed
+ */
+export const initializeServerTracing = () => {
+    try {
+        // Initialize the tracer provider
+        provider = new NodeTracerProvider({
+            resource: new Resource({
+                [SemanticResourceAttributes.SERVICE_NAME]: SERVICE_NAME
+            })
+        })
+
+        // Add B3 propagator
+        provider.addSpanProcessor(new SimpleSpanProcessor())
+        propagation.setGlobalPropagator(new B3Propagator())
+        provider.register()
+
+        return provider
+    } catch (error) {
+        // Log errors from OpenTelemetry initialization
+        console.warn('Failed to initialize OpenTelemetry provider:', error.message)
```

Comment:
Let's emit the log warnings using the PWAKitLogger to add a namespace. 

E.g.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/2c1e74d789026ff73b336abe0f62d23c93e4b1bb/packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js#L73